### DISCUSSION TITLE
Dynamically set Highlighted area

### DISCUSSION THREAD

**Discussion by dev-rish** [img-mapper/img-mapper#8] [Q&A] [answered]
Lets say i have list of all the areas and the ImaggerMapper component. Is it possible to dynamically select the area which is clicked in the list?

**Comment by NishargShah** [comment, marked as answer]
You can use the `onClick` callback function, it gives you the current area, index, and image event

```
onClick(area: obj, index: num, event): void
```

**Reply by dev-rish** [reply]
Could you please elaborate a bit more. I have this [sandbox](https://codesandbox.io/s/react-image-mapperissue-jpis4?file=/src/App.js). It has the example image and the list of all the areas in the image. I just want that whenever any item is clicked in the list corresponding area gets highlighted in the image. If you could update it then it would be great.

Thanks

**Reply by NishargShah** [reply]
You can use `preFillColor` for that

https://codesandbox.io/s/react-image-mapperissue-forked-nfxkb?file=/src/App.js

**Reply by gpanciera** [reply]
Would there be a way to do this but using the outline instead of fill? An example where I'd like to do this is in hovering over one area, I want to highlight related areas.
